locales/pt_BR.json

```diff
@@ -0,0 +1,777 @@
+			"fail": "A região {{=it.data.role }} está contida em outra região."
+		},
+		"page-has-heading-one": {
+			"pass": "A página tem pelo menos um título de primeiro nível",
+			"fail": "A página deve ter pelo menos um título de primeiro nível"
+		},
+		"page-has-main": {
+			"pass": "O documento tem pelo menos uma região 'main'",
+			"fail": "O documento não tem uma região 'main'"
+		},
+		"page-no-duplicate-banner": {
+			"pass": "O documento não tem mais de uma região 'banner'",
+			"fail": "O documento tem mais de uma região 'banner'"
+		},
+		"page-no-duplicate-contentinfo": {
+			"pass": "O documento não tem mais de uma região 'contentinfo'",
+			"fail": "O documento tem mais de uma região 'contenteinfo'"
+		},
+		"page-no-duplicate-main": {
+			"pass": "O documento não tem mais de uma região 'main'",
+			"fail": "O documento tem mais de uma região 'main'"
+		},
+		"tabindex": {
+			"pass": "O elemento não tem um 'tabindex' maior do que 0",
+			"fail": "O elemento tem um 'tabindex' maior do que 0"
+		},
+		"alt-space-value": {
+			"pass": "O elemento tem um valor válido para o atributo 'alt'",
+			"fail": "O elemento tem um atributo 'alt' contendo apenas um caracter de espaço, o que não é ignorado por todos os leitores de tela"
+		},
+		"duplicate-img-label": {
+			"pass": "O elemento não duplica um texto existente no atributo 'alt' do elemento <img>",
+			"fail": "O elemento contém o elemento <img> com texto alternativo duplicando um texto existente"
+		},
+		"explicit-label": {
+			"pass": "O elemento de formulário tem um <label> explícito",
+			"fail": "O elemento de formulário não tem um <label> explícito"
+		},
+		"help-same-as-label": {
+			"pass": "O texto de ajuda ('title' ou 'aria-describedby') não duplica o texto do 'label'",
+			"fail": "O texto de ajuda ('title' ou 'aria-describedby') é o igual ao texto do 'label'"
+		},
+		"hidden-explicit-label": {
+			"pass": "O elemento de formulário tem um <label> explícito visível",
+			"fail": "O elemento de formulário tem um <label> explícito que está oculto"
+		},
+		"implicit-label": {
+			"pass": "O elemento de formulário tem um <label> implícito (envolvido)",
+			"fail": "O elemento de formulário não tem <label> implícito (envolvido)"
+		},
+		"label-content-name-mismatch": {
+			"pass": "O elemento contém o texto visível como parte do seu nome acessível",
+			"fail": "O texto dentro do elemento não está incluído no nome acessível"
+		},
+		"multiple-label": {
+			"pass": "O campo de formulário não tem múltiplos elementos <label>",
+			"fail": "O uso de múltiplos elementos <label> não é amplamente suportado nas tecnologias assistivas"
+		},
+		"title-only": {
+			"pass": "Elementos de formulário não usam somente o atributo 'title' como seu rótulo",
+			"fail": "Apenas 'title' usado para gerar o rótulo do elemento de formulário"
+		},
+		"landmark-is-unique": {
+			"pass": "Os 'landmarks' devem ter um 'role' ou combinação de role/label/title (ou seja, nome acessível) únicos",
```

Comment:
Translate 'landmarks', 'role' and 'role/label/title'

Reply:
Like before: this are all atributes, and they will be used as they are (in English).